README.md

```diff
@@ -1,59 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Claude Mucyo Ntacyonungutse
 
 > One paragraph statement about the project.
 
-![screenshot](./app_screenshot.png)
+![screenshot](./app_screenshot.JPG)
 
 Additional description about the project and its features.
 
 ## Built With
 
 - Major languages
```

Comment:
- [ ] It would be fine if you remove this line because it is the direction for you. Instead of this, you should write the languages you used 😃.